test/azure/legacy/Expected/AcceptanceTests/CustomBaseUri/generated_samples/paths_get_empty.py

```diff
@@ -0,0 +1,30 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from custombaseurl import AutoRestParameterizedHostTestClient
+
+"""
+The sample just shows how to use the method and may not run successfully.
+# PREREQUISITES
+    pip install AutoRestParameterizedHostTestClient
+# USAGE
+    python paths_get_empty.py
+"""
+
+
+def main():
+    client = AutoRestParameterizedHostTestClient()
```

Comment:
We should also pass in the client parameters when initializing the client

Reply:
in sample we will only pass in required and no-default-value parameters. For this case, [host] has default value so it is not passed in: 
https://github.com/Azure/autorest.python/blob/d323963ea2328b1e6bd0b2ff4c377178c078db9b/test/azure/legacy/Expected/AcceptanceTests/CustomBaseUri/custombaseurl/_auto_rest_parameterized_host_test_client.py#L32